### DISCUSSION TITLE
A way to dump badges

### DISCUSSION THREAD

**Discussion by Xorrak32** [chromiecraft/chromiecraft#5808] [Suggestions]
So after you've got all the gear you want from the badge vendor, the badges become useless. I suggest an added way to dump it into usable things. 
One suggestion would be to be able to buy gems for badges, like in wrath with heroism emblems. 
Another suggestion would be for honor. 100 honor per badge or similarly.
Optionally a vendor for pets, toys, flares, rockets or the likes.
I think it would be a welcomed addition to many players.

**Comment by Aethereus-ManaOnion** [comment]
I think that honor would be a nice solution that works well in later expansions

**Comment by balleny** [comment]
A) there was nothing to dumb them into during this content phase on retail either.
B) there will be new gear in different stages available for the same badges in tbc content. badges wont reset between those stages, and it makes sense to gather some badges ahead of time because some items will cost up to 150 badges.
C) also, later there gonna be a badges sink in the form of epic gems for your gear.

tl;dr: just be patient

**Reply by balleny** [reply]
further, if you could e.g. convert badges into honor, the "what is left of pvp on chromiecraft" would be pushed even further down the drain.
pets and other cosmetic items are kind of reserved for the rewards catalog as of now, to reward contributions to the server (in the form of bug reporting, fixing things and donations)

**Reply by Xorrak32** [reply]
Chromie already has things that is not 100% blizzlike
Waiting for the epic gems will take at least 1 year, if not 2.

Everyone needs blue gems, If they put like a 20-30 badge per gem price, I don't think the AH would be flooded by them. 
PvP people complain that people don't have PvP gear when joining. This would give newcomers a begining to gear up. I don't think anyone would complain about it to be honest. Besides, this is already a thing in wotlk. I know this isn't exactly like retail tbc, but like I said, chromie has many things retail didn't have already. It would give an incentive to keep doing dungeons also.

**Reply by Aethereus-ManaOnion** [reply]
All I can hear is that the game design of content made in 2007 wasn't perfect, so I ultimately agree
